packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -261,6 +262,8 @@ class _HeroState extends State<Hero> {
 
   @override
   Widget build(BuildContext context) {
+    assert(context.ancestorWidgetOfExactType(Hero) == null, 'A Hero widget cannot be the descendant of another Hero widget.');
```

Comment:
this line should probable be line broken:

```suggestion
    assert(
      context.ancestorWidgetOfExactType(Hero) == null,
      'A Hero widget cannot be the descendant of another Hero widget.'
    );
```